Packs/OTSecurity/IndicatorTypes/reputation-MAC_Address.json

```diff
@@ -1,6 +1,7 @@
 {
   "regex": "^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$",
   "details": "MAC Address",
+  "name": "SOME NAME",
```

Comment:
```suggestion
```
this will be fixed soon.
anyway, this error is not raised so you can simply ignore it now.